WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiHttpApiIntegrationTests.cs

```diff
@@ -103,11 +104,16 @@ public async Task SoloCoinJoinTest()
 
 			// Create the coinjoin client
 			var apiClient = _apiApplicationFactory.CreateWabiSabiHttpApiClient(httpClient);
+
 			var rounds = await apiClient.GetStatusAsync(CancellationToken.None);
 			var roundState = rounds.First(x => x.CoinjoinState is ConstructionState);
 			var kitchen = new Kitchen();
 			kitchen.Cook("");
-			using var coinJoinClient = new CoinJoinClient(roundState.Id, apiClient, coins, kitchen, keyManager);
+
+			using var roundStateUpdater = new RoundStatusUpdater(TimeSpan.FromSeconds(1), apiClient);
+			await roundStateUpdater.StartAsync(CancellationToken.None);
```

Comment:
We have to stop it at the end of the test.

Reply:
ACK